scripts/build_utils.sh

```diff
@@ -592,6 +597,8 @@ use_ppa() {
                         use_ppa=false
                     fi
                     ;;
+                focal)
+                    use_ppa=true;;
```

Comment:
drop this change, as focal comes with gcc-9 which is good enough for us at this moment.